<launch>
    <node name = 'move_robot_node' pkg = 'collision_predictor' type = 'move.py' output = 'screen'></node>
</launch>